modules/text_server_adv/SCsub

```diff
@@ -469,17 +474,26 @@ if env["builtin_icu"]:
             "-DUCONFIG_NO_IDNA",
             "-DUCONFIG_NO_FILE_IO",
             "-DUCONFIG_NO_TRANSLITERATION",
-            "-DPKGDATA_MODE=static",
             "-DICU_DATA_NAME=" + icu_data_name,
         ]
     )
+    if not env["shared"]:
+        env_icu.Append(
+            CXXFLAGS=[
+                "-DU_STATIC_IMPLEMENTATION",
+                "-DPKGDATA_MODE=static",
```

Comment:
Data mode probably should be static for the "shared" build as well, since data is not built as a separate icudata library, (which is only supported on Windows, if I remember correctly).

Reply:
Yes, thanks for looking! I made have been presumptuous about the word "static" in this define. I will push that define back into the common set.

Reply:
fixed in latest push